build/CoreFoundation/dll/CoreFoundation.vcxproj

```diff
@@ -152,7 +152,7 @@
       <IgnoreAllDefaultLibraries>false</IgnoreAllDefaultLibraries>
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
       <ModuleDefinitionFile>CoreFoundation.def</ModuleDefinitionFile>
-      <AdditionalDependencies>mincore.lib;libxml2.lib;icudt.lib;icuin.lib;icuuc.lib;libdispatch.lib;icudata.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
Net increase in size, probably because more complete data set now? 

---
In reply to: [71421348](https://github.com/Microsoft/WinObjC/pull/668#discussion_r71421348) [](ancestors = 71421348,71421042,71389811)